Automattic/wp-calypso
# A4A: Implement reusable hosting sections (Features, Testimonials and Additional Features)

This PR implements a couple of hosting section reusable components.

**Additional Features section**
<img width="1435" alt="Screenshot 2024-07-25 at 2 41 15 PM" src="https://github.com/user-attachments/assets/29177d2e-0d31-49f4-bdbf-f59dd94a1e70">

**Features section**
<img width="1444" alt="Screenshot 2024-07-25 at 2 41 24 PM" src="https://github.com/user-attachments/assets/e7855158-2d71-45f8-8832-087a2450a462">

**Testimonial section**
<img width="1445" alt="Screenshot 2024-07-25 at 2 41 31 PM" src="https://github.com/user-attachments/assets/37ea3ecb-14e8-4b3c-9aa0-1c45fec21cf8">


Depends on https://github.com/Automattic/wp-calypso/pull/92943

Closes https://github.com/Automattic/jetpack-genesis/issues/429
Closes https://github.com/Automattic/jetpack-genesis/issues/431
Closes https://github.com/Automattic/jetpack-genesis/issues/432

## Proposed Changes

* Implement the `HostingFeaturesSection` based on the `HostingSection` and `HostingOverviewFeatures` components.
* Implement the `HostingTestimonialsSection` based on the `HostingSection`.
* Implement the `HostingAdditionalFeaturesSection` based on the `HostingSection`.
* Add sample usage for easier testing.

## Testing Instructions

* Use the A4A live link and go to `/marketplace/hosting`
* Confirm that the Hosting sections are visible and correct.

## Pre-merge Checklist
